Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -153,10 +162,15 @@ const SubmissionDetails = () => {
 
             setSubmissionDetailsResultsUrlParams({
                 submissionId,
-                take: DEFAULT_PROBLEM_RESULTS_TAKE_CONTESTS_PAGE,
+                page: currentPage,
             });
         },
-        [ currentSubmission, setSubmissionDetailsResultsUrlParams ],
+        [ currentPage, currentSubmission, setSubmissionDetailsResultsUrlParams ],
```

Comment:
Adding page in the url and reusing the same Action in the SubmissionsController results in taking not only the results but also the details for the submissions, which are not changed. Maybe it is better to add another Action that will be responsible for getting the submission results. @gogo4ds  @shsimeonova  Do you agree ?

Reply:
Refactored